shared/search/user-pane/non-user.render.js.flow

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+
+import {Component} from 'react'
+
+export type Props = {
+  avatar: ?string,
+  username: string,
+  fullname?: string,
+  serviceName: string,  // FIXME (MBG): do we have a type for this?
```

Comment:
Do we have a flowtype def for the complete set of possible service names? I went looking for one but didn't find it.